tools/query_verification/src/main/java/com/google/bigquery/QueryVerifier.java

```diff
@@ -112,5 +117,43 @@ public void verifyDataAware() throws InterruptedException {
         System.out.println("Data-Aware Verification Completed");
     }
 
+    /**
+     * Finds extra and missing results by locating the differences between the results.
+     * @param migratedResults Parsed results returned from BQ
+     * @param originalResults Parsed results returned from original data warehouse service
+     * @return Differences classified as either extra or missing from migrated results.
+     */
+    public ResultDifferences compareResults(List<QueryJobResults> migratedResults, List<QueryJobResults> originalResults) {
+        // Check if same amount of queries were run
+        if (migratedResults.size() == originalResults.size()) {
```

Comment:
Why does this have to be an if condition? I would assume we'd want to return information around differences in row counts etc.

Reply:
This is actually just checking if the same number of query statements were run. If they are not equal, that means either `migratedResults` or `originalResults` has the results of queries that aren't in the other and it'd be difficult to determine which ones those are

Reply:
I see, makes sense then. Thanks!